googletest/CMakeLists.txt

```diff
@@ -11,15 +11,26 @@ FetchContent_Declare(
 FetchContent_MakeAvailable(googletest)
 
 list(APPEND dxlibs "")
-if(EXISTS "/dev/dxg" AND EXISTS "/lib/wsl/lib/")
-  find_library(libd3d12 d3d12 HINTS /lib/wsl/lib)
-  find_library(libdxcore dxcore HINTS /lib/wsl/lib)
-  list(APPEND dxlibs ${libd3d12} ${libdxcore})
+if(EXISTS "/usr/lib/wsl/lib/")
+  find_library(libd3d12 d3d12 HINTS /usr/lib/wsl/lib)
```

Comment:
@jenatali It looks like this was set up for a preview scenario, but in Windows 11 WSL (at least with Ubuntu), the libraries are actually in ``/user/lib/wsl/lib``.

Reply:
Oh, you're totally right. I dunno how it ever worked before, unless maybe the hint prepended `/usr` to the search path automatically?